samza-kv/src/main/scala/org/apache/samza/storage/kv/LoggedStore.scala

```diff
@@ -24,8 +24,8 @@ import org.apache.samza.system.{OutgoingMessageEnvelope, SystemStreamPartition}
 import org.apache.samza.task.MessageCollector
 
 /**
- * A key/value store decorator that adds a changelog for any changes made to the underlying store
- */
+  * A key/value store decorator that adds a changelog for any changes made to the underlying store
```

Comment:
Please revert indentation changes from javadocs in all classes, unless this is what IntelliJ does by default.

Reply:
Looks like all changes in this class can be reverted?

Reply:
These changes were done by IntelliJ by default hence not reverted back.